compiler/bindings/python/iree/compiler/tools/tflite.py

```diff
@@ -68,67 +68,6 @@ class ImportOptions(CompilerOptions):
   input_type: Optional[str] = "tosa"
 
 
-def build_import_command_line(input_path: str, tfs: TempFileSaver,
-                              options: ImportOptions) -> List[str]:
-  """Builds a command line for invoking the import stage.
-
-  Args:
-    input_path: The input path.
-    tfs: TempFileSaver.
-    options: Import options.
-  Returns:
-    List of strings of command line.
-  """
-  import_tool = find_tool(_IMPORT_TOOL)
-  cl = [
-      import_tool,
-      input_path,
-  ]
-
-  if options.import_only and options.output_file:
-    # Import stage directly outputs.
-    output_file = tfs.alloc_optional("tflite-output.mlir",
-                                     export_as=options.output_file)
-    cl.append(f"-o={options.output_file}")
-
-  # Input arrays.
-  if options.input_arrays:
-    for input_array in options.input_arrays:
-      cl.append(f"--input-array={input_array}")
-    for output_array in options.output_arrays:
-      cl.append(f"--output-array={output_array}")
-
-  # MLIR flags.
-  if options.output_mlir_debuginfo:
-    cl.append("--mlir-print-debuginfo")
-  if options.output_generic_mlir:
-    cl.append("--mlir-print-op-generic")
-
-  # Save temps flags.
-  tfl_input = tfs.alloc_optional("tflite-input.mlir",
-                                 export_as=options.save_temp_tfl_input)
-  if tfl_input:
-    cl.append(f"--save-temp-tfl-input={tfl_input}")
-  iree_input = tfs.alloc_optional("tflite-iree-input.mlir",
-                                  export_as=options.save_temp_iree_input)
-  if iree_input:
-    cl.append(f"--save-temp-iree-input={iree_input}")
```

Comment:
Are these options still supported? Might take changes like those in https://github.com/openxla/iree/pull/13287